presto-spi/pom.xml

```diff
@@ -53,6 +53,18 @@
             <artifactId>jol-core</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
We do not want to add these dependencies to the SPI.  Guava in particular is heavily used and likely to cause conflicts on the developers who need to bring it in.  Please refactor to avoid these dependencies.

Reply:
I would keep `RequestInfo` where it is, but create a new DTO in the SPI that we can translate to/from.